### PR TITLE
✨ new API entrypoint `web-server:clone_project` and drafts `api-server:clone_study`

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- [x] new route to `clone_project` in `web-server`
   - refactoring since `File _crud_handlers.py has 501 lines of code (exceeds 500 allowed).` 
- [x] exposes  entrypoint for `clone_study` in `api-server`. 
    - For the sake of clarity, the implementation connecting both entrypoints will be in next PR

## Related issue/s

- implementing #4177
- implementing https://github.com/ITISFoundation/osparc-simcore/issues/4651

## How to test

Driving test in the webserver are
```
cd services/web/server
pytest -vv tests/unit/with_dbs/02/test_projects_ports_handlers.py::test_clone_project_and_set_inputs
pytest -vv tests/unit/with_dbs/02/test_projects_ports_handlers__clone.py::test_clone_project
```

 
## DevOps 

None

### CODE REVIEW

File: api/specs/web-server/_projects_crud.py [hunk 1]
```diff
@@ -39,19 +39,14 @@
...
-
 @router.post(
     "/projects",
     response_model=Envelope[TaskGet],
     summary="Creates a new project or copies an existing one",
     status_code=status.HTTP_201_CREATED,
 )
 async def create_project(
-    _params: Annotated[_ProjectCreateParams, Depends()],
+    _params: Annotated[ProjectCreateParams, Depends()],
```
Comment (GitHK) [inline api/specs/web-server/_projects_crud.py:49]: Q: you add it via depends so you don't have to specify each individual parameter one by one and this is kept into sync? Did I get it correct?

**Review by GitHK — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]
nice!

File: api/specs/web-server/_projects_crud.py (hunk 1 above)
Comment (pcrespov) [reply api/specs/web-server/_projects_crud.py:49]: yes

File: packages/service-library/src/servicelib/aiohttp/long_running_tasks/client.py [hunk 2]
```diff
@@ -91,6 +84,7 @@ async def _task_result(session: ClientSession, result_url: URL) -> Any:
             assert not error  # nosec
             assert data  # nosec
             return data
+        return None
```
[the remark this answers is not in the document]
Comment (pcrespov) [reply packages/service-library/src/servicelib/aiohttp/long_running_tasks/client.py:87]: https://beta.ruff.rs/docs/rules/implicit-return/

**Pull request merged**